notebooks/environment.yml

```diff
@@ -0,0 +1,8 @@
+name: landlab_notebooks
+channels:
```

Comment:
I'm almost certain that it makes sense to get rid of this file and the associated req.txt in the notebook folder. 

Binder needs a file called environment.yml at the top level directory (almost certain of this... hard to check since binder is being kinda funny today)

We also have a somewhat complex binder config b/c we want an environment AND we want setup.py to get run.

Reply:
I would be surprised if *Binder* doesn't allow you to specify a configuration file of your choice. However, if it doesn't, we can just move this up a level.

I was thinking we ***don't*** want to run *setup.py* for *Binder*. My thought was that the *Binder* examples would be for users wanting to use the latest landlab release - not the development version.

Reply:
Binder has a lot of configuration file options (https://mybinder.readthedocs.io/en/latest/config_files.html#postbuild-run-code-after-installing-the-environment) that it considers in a relatively standard order. 

I'm not aware of a way to indicate which environment file one wants to use, just that if you want the most common approach, you need to have a file called environment.yml at the top level. 

I think you are right about using the release version of landlab. that makes sense.